kfdefs/overlays/moc/smaug/opf-kafka/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-kafka
+
+resources:
+  - ../../../../base/kafka
+patchesJson6902:
+  - patch: |
+      - op: add
+        path: /spec/applications/-
+        value:
+          kustomizeConfig:
+            overlays:
+              - topics
+              - users
+            repoRef:
+              name: opf
+              path: odh-manifests/smaug/kafka
```

Comment:
we will need to make these changes in the `odh-manifests` directory

Reply:
PR for the odh-manifests directory: https://github.com/operate-first/apps/pull/1021